frontends/election-manager/src/config/types.ts

```diff
@@ -40,6 +40,7 @@ export enum BallotMode {
   Official = 'live',
   Test = 'test',
   Sample = 'sample',
+  Draft = 'draft',
```

Comment:
Maybe some comments explaining these modes could be helpful at this point. I'm starting to get confused 😄

Reply:
Excellent call! Added in https://github.com/votingworks/vxsuite/pull/2249/commits/7520221ef50d53b0d2d7b15b81b2f71de47be2cf